build/Targets/GenerateCompilerExecutableBindingRedirects.targets

```diff
@@ -5,7 +5,7 @@
   <PropertyGroup>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
   </PropertyGroup>
-
+  
```

Comment:
@Pilchie Maybe I am confusing myself. This is for XML, and I been thinking C# (and JS/HTML) which VS trims trailing whitespace whenever a file is saved. (the/a whitespace plugin shows that behavior). My gripe still stands, any good editor should strip trailing whitespace on save unless it is explicitly demanded by the code/language of the file being edited.